backend/src/main/java/com/redhat/cloud/notifications/routers/models/behaviorgroup/CreateBehaviorGroupRequest.java

```diff
@@ -35,4 +40,15 @@ public class CreateBehaviorGroupRequest {
      * the behavior group).
      */
     public Set<UUID> eventTypeIds;
+
+    /**
+     * Validates that the bundle is identifiable.
+     * @return true if the bundle ID is null and the bundle name is blank. False otherwise.
```

Comment:
```suggestion
     * @return true if the bundle ID is not null or the bundle name is not blank. False otherwise.
```

Reply:
Phew! On this one I've changed the name of the function so that it better reflects what it is supposed to do too, because with the change in the Javadocs the returning result wasn't being honored by what the function was doing...